README.adoc

```diff
@@ -285,6 +285,102 @@ spec:
 ...
 ----
 
+== Configure Health
+
+=== Kiali defaults
+
+By default Kiali health is:
+
+* Degraded when Error Rate is >= 0.1%
+* Failure when Error Rate is >= 20%
+
+For Kiali an error request is when:
+
+* For http the code status match the expression */^[4-5]\d\d$/*
+* For grpc the code status match the expression */^ [1-9]$|^1[0-6]$/*
+
+=== Configuration
+
+Kiali works with a FIFO configuration, so Kiali will use the first configuration that applies with the resource requested.
+
+So the granularity and criteria should be configured from highest to lowest.
+
+The health configuration follow this scheme:
+
+[source,yaml]
+----
+health_config:
+  rate:
+    - namespace: string
+      kind: string
+      name: string
+      tolerance:
+        - protocol: string
+          direction: string
+          code: string
+          degraded: number
+          failure: number
+          ...
+    ...
+
+
+----
+
+For each Rate Kiali will use the first match by:
+
+* Namespace (string) is a regular expression that match the namespace name (bookinfo, istio-system...)
+* Kind (Optional) (string) is a regular expression that match the resource type (app/workload/service)
+* Name (Optional) (string) is a regular expression that match the resource name (reviews, productpage...)
+
+If a configuration match kiali will use the first tolerance configuration that match:
```

Comment:
It would be good to indicate user where these matching fields are coming from.
It implicitly requires that user understand that these are istio collected metrics.
It would be worth to add better description here.